src/contrib/persistence/Akka.Persistence.Query.Sql/AllPersistenceIdsPublisher.cs

```diff
@@ -175,16 +175,12 @@ private bool Active(object message)
         {
             switch (message)
             {
-                case CurrentPersistenceIds added:
-                    _lastOrderingOffset = added.HighestOrderingNumber;
-                    _buffer.AddRange(added.AllPersistenceIds);
-                    _buffer.DeliverBuffer(TotalDemand);
-                    return true;
                 case Request _:
                     _buffer.DeliverBuffer(TotalDemand);
                     return true;
                 case Continue _:
                     _journalRef.Tell(new SelectCurrentPersistenceIds(_lastOrderingOffset, Self));
+                    Become(Waiting);
```

Comment:
With short `refresh-interval`, we will be spamming the back-end with sql queries, this will throttle the refresh frequency to the speed of the back-end